### PR TITLE
Add server manager hint explanation

### PR DESCRIPTION
**Pull request by doctaweeks**

### CODE REVIEW

File: Writerside/topics/Server-Manager.md [hunk 1]
```diff
@@ -1,49 +1,103 @@
...
+to run matches.
 
-An example of the simple flow, running a single match, is included in a [mock game server script](https://github.com/CatenaTools/catena-tools-core/blob/main/mocks/gameserver.py). 
+An example of the simple flow, running a single match, is included in
+a [mock game server script](https://github.com/CatenaTools/catena-tools-core/blob/main/mocks/gameserver.py).
 
 ### `RequestMatch`
 
-When a game server is idle (or backfilling or running matches in parallel) and has room for a new match it can call `RequestMatch()`. This will request a match from a [match broker](Match-Broker.md), optionally meeting any requirements of the game server by including a <tooltip term="JMESPath">JMESPath</tooltip> expression. The result of this may be one match or that there are no matches are available meeting the requirements in which case the game server may want to periodically retry until it gets a match or just exit (user implementation specific). This first use of this call is also the first indication to the backend that a server is extant.
+When a game server is idle (or backfilling or running matches in parallel) and has room for a new match it can
```
Comment (jlyon1) [inline Writerside/topics/Server-Manager.md:29]: For this section and the other requests, lets pull in the openapi docs now that they are being generated. They are located in this directory: https://github.com/CatenaTools/documentation/tree/main/Writerside/apispec/openapi

Comment (jlyon1) [reply Writerside/topics/Server-Manager.md:29]: They can be under a collapsible section so they aren't the main draw.

**Review by jlyon1 — CHANGES_REQUESTED** [review]
Minor recommendation

**Comment by hunterfortuin** [comment]
Not necessarily directly related to this review, but something that is related to a topic brought up in #9 and was added [here](https://github.com/CatenaTools/documentation/pull/9/commits/05e2f7565df397e794af6b0f3c962f51b683b9a2#diff-08e2dc4d4101c1b6c8eac3f22f43d0f3d579e6acc686be9c2839b986a49c3d31R67).

I think it would be helpful to show some diagrams showing how each of these components fit together (game client, matchmaker, match broker, server manager, etc) so readers can more easily visualize the system.

**Comment by doctaweeks** [comment]
> Not necessarily directly related to this review, but something that is related to a topic brought up in #9 and was added [here](https://github.com/CatenaTools/documentation/pull/9/commits/05e2f7565df397e794af6b0f3c962f51b683b9a2#diff-08e2dc4d4101c1b6c8eac3f22f43d0f3d579e6acc686be9c2839b986a49c3d31R67).
> 
> I think it would be helpful to show some diagrams showing how each of these components fit together (game client, matchmaker, match broker, server manager, etc) so readers can more easily visualize the system.

Good call - started here with the server manager: https://github.com/CatenaTools/documentation/pull/19

I will do another for the integrated match broker.

File: Writerside/topics/Server-Manager.md (hunk 1 above)
Comment (jlyon1) [reply Writerside/topics/Server-Manager.md:29]: We can address this in a later standardization effort

**Review by jlyon1 — APPROVED** [review]

**Pull request merged**